src/color.h

```diff
@@ -181,10 +210,10 @@ nc_color magenta_background(nc_color c);
 nc_color cyan_background(nc_color c);
 nc_color rand_color();
 
-nc_color color_from_string(std::string const &color);
-std::string const& string_from_color(nc_color color);
-nc_color bgcolor_from_string(std::string const &color);
-nc_color get_color_from_tag(const std::string &s, nc_color base_color);
+nc_color color_from_string(const std::string  &color);
+std::string const& string_from_color(const nc_color &color);
+nc_color bgcolor_from_string(const std::string  &color);
+nc_color get_color_from_tag(const std::string &s, const nc_color base_color);
```

Comment:
I think these changes are pointless.

The `const` in `const nc_color base_color` is definitely pointless, see http://stackoverflow.com/questions/17208570/top-level-const-doesnt-influence-a-function-signature, it's ignored here anyway

`void f( int x );` and `void f( const int );` define exactly the same function, they are not overloaded functions.